Source/gg2/Scripts/Killtable/killtable_clear.gml

```diff
@@ -0,0 +1,2 @@
+//Clears the given killtable
+argument0 = killtable_create();

```

Comment:
You can't replace the contents of a variable for the caller, because that would be call-by-reference and GM does not have that. All you are doing is setting the local variable called argument0 to a new killtable, and that variable will stop existing after the script returns, so nobody will ever see (or destroy) the new killtable again, and the caller will not see any effect at all.

Don't bother fixing it though, I'm replacing the killtable with a ds_map. You should look at that structure, most programming languages offer something like it.

Reply:
Ok wow looks like the previous code used 3 ds_lists instead of a ds_map, much cleaner, I agree